src/components/button/_macro.njk

```diff
@@ -2,22 +2,48 @@
+  {% elif params.buttonStyle == "mobile" %}
+    {% set iconType = "chevron-down" %}
+    {% set iconPosition = "after" %}
+  {% endif %}
+{% endif %}
+
   <{{ tag }}
       {% if params.url is defined %}
         href="{{ params.url }}"
         role="button"
       {% else %}
-        type="{{ params.type if params.type else ('button' if params.print else 'submit') }}"
+        type="{{ params.type if params.type is defined else ('button' if params.buttonStyle == "print" else 'submit') }}"
       {% endif %}
-      class="btn{% if params.classes is defined %} {{ params.classes }}{% endif %}{% if params.url is defined and not params.hideCaret %} btn--link{% endif %}{% if params.disabled is defined %} btn--disabled{% endif %}{% if params.print is defined %} btn--print u-d-no js-print-btn{% endif %}{% if params.exit is defined %} btn--exit{% endif %}{% if params.submitType == "loader" %} btn--loader js-loader js-submit-btn{% endif %}{% if params.submitType == "timer" %} js-timer js-submit-btn{% endif %}"
+      class="btn{% if params.classes is defined %} {{ params.classes }}{% endif %}{% if params.url is defined and not params.hideCaret %} btn--link{% endif %}{% if params.disabled is defined %} btn--disabled{% endif %}{% if params.buttonStyle == "print" %} btn--print u-d-no js-print-btn{% endif %}{% if params.buttonStyle == "exit" %} btn--exit{% endif %}{% if params.submitType == "loader" %} btn--loader js-loader js-submit-btn{% endif %}{% if params.submitType == "timer" %} js-timer js-submit-btn{% endif %}"
       {% if params.id is defined %}id="{{ params.id }}"{% endif %}
       {% if params.value is defined %}value="{{ params.value }}"{% endif %}
       {% if params.name is defined %}name="{{ params.name }}"{% endif %}
       {% if params.disabled is defined %} disabled{% endif %}
       {% if params.url is defined and params.newWindow is defined %}target="_blank" rel="noopener"{% endif %}
       {% if params.attributes is defined %}{% for attribute, value in (params.attributes.items() if params.attributes is mapping and params.attributes.items else params.attributes) %}{{attribute}}="{{value}}" {% endfor %}{% endif %}
       >
-      <span class="btn__inner{% if params.innerClasses is defined %} {{ params.innerClasses }}{% endif %}{% if params.submitType != 'loader' %}{% if params.newWindow is defined %} icon--external-link {% elif params.exit is defined %} icon--exit {% endif %}{% endif %}">{{- params.html | safe if params.html else params.text -}}</span>
+      <span class="btn__inner{% if params.innerClasses is defined %} {{ params.innerClasses }}{% endif %}">
+        {% if iconPosition == "before" %}<svg><use xlink:href="/img/icons/icons--{{ iconType }}.svg#{{ iconType }}"></use></svg>{% endif %}
+        {{- params.html | safe if params.html is defined else params.text -}}
+        {% if iconPosition == "after" %}<svg><use xlink:href="/img/icons/icons--{{ iconType }}.svg#{{ iconType }}"></use></svg>{% endif %}
```

Comment:
We were discussing on the DS stand up about making the path to icons configurable (so you could have a brand pack of icons). Thinking we could set the path at the `pageConfig` level and have a default.

Reply:
Yeah ok ill sort that